### DISCUSSION TITLE
can not upload files

### DISCUSSION THREAD

**Discussion by tony-is-coding** [fastapi/fastapi#8196] [Questions] [answered]
i want to upload a  image file,  here is my code 
![image](https://user-images.githubusercontent.com/34122761/61853876-7bdcde80-aeef-11e9-8ecc-c2ccd9803f59.png)
i has try to upload it in swagger and post man ,both got a 422 error: There was an error parsing the body
![image](https://user-images.githubusercontent.com/34122761/61854141-10dfd780-aef0-11e9-8f76-edd596e2ee00.png)
![image](https://user-images.githubusercontent.com/34122761/61854180-205f2080-aef0-11e9-81ab-93d831ac52b6.png)

please give me help!

**Comment by tiangolo** [comment]
I assume you were able to solve your problem. Thanks for closing the issue.

Feel free to add more comments or new issues.

**Comment by jeremytanjianle** [comment]
How was the issue solved? I'm experiencing the same problem.

**Comment by tony-is-coding** [comment]
> How was the issue solved? I'm experiencing the same problem.

i had write a demo application: https://github.com/tony-is-coding/file_upload  hope can give some help to you

**Comment by pranithan-kang** [comment, marked as answer]
More information for the fresh project, you have to install `python-multipart`. Refer to [this](https://stackoverflow.com/a/62420332/2003339) and [this](https://gitter.im/tiangolo/fastapi?at=5e41a833c900d747abf806d3) links.
